app/uk/gov/hmrc/gform/formtemplate/FormComponentMaker.scala

```diff
@@ -138,6 +138,9 @@ class FormComponentMaker(json: JsValue) {
   lazy val optInstruction: Opt[Option[Instruction]] =
     toOpt((json \ "instruction").validateOpt[Instruction], "/instruction")
 
+  lazy val chooseAddressLabel: Option[SmartString] = (json \ "chooseAddressLabel").asOpt[SmartString]
+  lazy val confirmAddressLabel: Option[SmartString] = (json \ "confirmAddressLabel").asOpt[SmartString]
```

Comment:
We need to stop using `.asOpt[SmartString]` as it is not safe. Try for example

```
"chooseAddressLabel": "Test ${1 + + 2}"
```

This will successfully upload, but `chooseAddressLabel` will not be set.

This is how I've fixed `helpText` in other PR:

```diff
-  lazy val helpText: Option[SmartString] = (json \ "helpText").asOpt[SmartString]
+  lazy val optHelpText: Opt[Option[SmartString]] = toOpt((json \ "helpText").validateOpt[SmartString], "/helpText")
```